src/game/Sprite.java

```diff
@@ -22,9 +22,9 @@ public Rectangle getBoundingBox() {
 	}
 
 	public Image getCachedImage(int nw, int nh) {
-		if (cache.containsKey(new Point(nw, nh))) {
-			return cache.get(new Point(nw, nh));
-		}
+		// if (cache.containsKey(new Point(nw, nh))) {
+		// return cache.get(new Point(nw, nh));
+		// }

```

Comment:
I would delete this instead of commenting it out

Reply:
Not sure if caching images is a good idea. Mby profile it?